tests/test_backend.py

```diff
@@ -512,6 +513,8 @@ def test_file(tmpdir, src_path, dst_path, version, backend, owner):
     assert os.path.exists(src_path)
     assert backend.checksum(dst_path, version) == audeer.md5(src_path)
     assert backend.owner(dst_path, version) == owner
+    date = datetime.datetime.today().strftime('%Y/%m/%d')
+    assert backend.date(dst_path, version) == date
```

Comment:
It means tests can fail if executed just before midnight, but this should not happen very often :)

Reply:
Should indeed be fine